contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -431,6 +443,8 @@
       fieldRequiredMessage: 'Field is required',
       errorsMessage: 'Please fix the errors below',
       registrationFailed: 'There was an error registering your account. Please try again',
+      registrationFailedOffline:
+        'You seem to be offline. Please connect to the internet before finishing registration.',
```

Comment:
We use `create an account` instead of `register`, so I'm wary of introducing _finishing registration_ here...? 🤔 

How about (cc @khangmach):
`You seem to be offline. Please connect to the internet to create an account`.

Reply:
the string revision here looks good to me